python/README.md

```diff
@@ -0,0 +1,20 @@
+# Build Instructions
```

Comment:
The wording `build` seems to general here. It might be better to mention what this section is building in the heading.

Reply:
Updated it to just "Instructions", lmk if that works.